interactions/models/internal/active_voice_state.py

```diff
@@ -289,15 +289,13 @@ async def _voice_server_update(self, data) -> None:
 
     async def _voice_state_update(
         self,
-        before: Optional[VoiceState],
         after: Optional[VoiceState],
         data: Optional[VoiceStateData],
     ) -> None:
         """
         An internal receiver for voice server state events.
 
         Args:
-            before: The previous voice state
```

Comment:
I'm sure changing around internals isn't breaking, but just in case another contributor hops it, I see no reason that this removal is needed, per se 😅.

Reply:
gotcha I reverted the change